tests/contrib/pyramid/test_pyramid.py

```diff
@@ -45,4 +45,4 @@ def test_distributed_tracing(self):
         eq_(span.trace_id, 100)
         eq_(span.parent_id, 42)
         eq_(span.get_metric('_sampling_priority_v1'), 2)
-        eq_(span.get_tag('_dd_origin'), 'synthetics')
+        eq_(span.get_tag('_dd.origin'), 'synthetics')
```

Comment:
We should likely use `from ddtrace.constants import ORIGIN_KEY` here, since we need to update `_dd_origin` in a few places now as well.